packages/component/src/SendBox/SuggestedActions.js

```diff
@@ -98,6 +98,7 @@ const SuggestedActions = ({ className, suggestedActions = [] }) => {
         dir={direction}
         flipperBlurFocusOnClick={true}
         showDots={false}
+        showScrollBar={false}
```

Comment:
Today, we show a white semi-transparent scroll bar. It think we should hide it.

![image](https://user-images.githubusercontent.com/1622400/82601286-2ea7b880-9b64-11ea-9221-789b7ed3ca06.png)

Reply:
Yea I'm good with that.